### PR TITLE
feat: create country field story

### LINKED ISSUE #481: Country field
basically a dropdown with country list filled in.

useful for
- citizenship / nationality
- place of birth
- component of address field

### PR DESCRIPTION
**Pull request by jia1**
## Problem

- Create country field story

Partially solves #481

## Solution

- Rely on `Dropdown` to create a `CountryField` component in `.stories`

**Breaking Changes** 

- [ ] Yes - this PR contains breaking changes
    - Details ...
- [x] No - this PR is backwards compatible  

## Before & After Screenshots

**AFTER**:

Screenshot 2022-04-05 at 6 23 57 PM

### CODE REVIEW

File: shared/constants/countries.ts [hunk 1]
```diff
@@ -0,0 +1,256 @@
+export enum Country {
+  AFGHANISTAN = 'AFGHANISTAN',
```
Comment (karrui) [inline shared/constants/countries.ts:2]: nit: keys should be SentenceCase, (values can remain as is)

e.g.
```ts
export enum Country {
  Afghanistan = 'AFGHANISTAN',
  ...
}
```

PS: Probably don't need to manually do it, can probably do it in a javascript/node playground haha, by 
1. converting the enum into a `object`, 
2. then running some sort of key-change function on the object, then 
3. JSONify the object,
4. copy paste back into this enum

**Review by karrui — APPROVED** [review]
lgtm

File: frontend/src/templates/Field/CountryField/CountryField.tsx [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  BasicField,
+  DropdownFieldBase,
+  FormFieldWithId,
+} from '~shared/types/field'
+
+import DropdownField from '~templates/Field/Dropdown'
+
+import { BaseFieldProps } from '../FieldContainer'
+
+export type CountryFieldSchema = FormFieldWithId<DropdownFieldBase>
```
[the remark this answers is not in the document]
Comment (jia1) [reply frontend/src/templates/Field/CountryField/CountryField.tsx:13]: Addressed by the latest commit `5d1da632a04966ad5c89153086406a78bca4f807`

File: shared/constants/countries.ts (hunk 1 above)
Comment (jia1) [reply shared/constants/countries.ts]: Addressed by the latest commit `5d1da632a04966ad5c89153086406a78bca4f807`

**Pull request merged**